drivers/scmi-msg/scmi_pinctrl.c

```diff
@@ -0,0 +1,890 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2024, EPAM Systems
+ */
+#include <cdefs.h>
+#include <string.h>
+#include <errno.h>
+
+#include <drivers/pinctrl.h>
+#include <drivers/scmi-msg.h>
+#include <drivers/scmi.h>
+#include <lib/utils.h>
+#include <lib/utils_def.h>
+
+#include "common.h"
+
+#pragma weak pinctrl_get_groups_count
```

Comment:
I believe ARM TF coding style requires those pragmas to be near the function definition

Reply:
UPD. I've checked - seems all weaks in ARM TF defined this way.